layouts/partials/article_metadata.html

```diff
@@ -45,8 +57,8 @@
   <span class="article-categories">
     <i class="fa fa-folder"></i>
     {{ range $k, $v := $.Params.categories }}
-    <a href="{{ ($.Site.GetPage (printf "categories/%s" .)).Permalink }}">{{ . }}</a
-    >{{ if lt $k (sub $categoriesLen 1) }}, {{ end }}
+    <a href="{{ ($.Site.GetPage (printf "categories/%s" .)).Permalink }}">{{ . }}</a>
```

Comment:
It was coded like this for a reason - your way introduces extra spacing in the output. Either leave it as it is or tell the Go templating to remove whitespace around the HTML tags with `{{- if lt $k (sub $categoriesLen 1) -}}, {{ end }}` (note the `{{-`).